src/app/Recordings/ActiveRecordingsTable.tsx

```diff
@@ -302,13 +328,31 @@ export const ActiveRecordingsTable: React.FunctionComponent<ActiveRecordingsTabl
           <Td key={`active-table-row-${props.index}_5`} dataLabel={tableColumns[3]}>
             {props.recording.state}
           </Td>
+          <Td key={`active-table-row-${props.index}_6`} dataLabel={tableColumns[4]}>
+            {editingMetadata ? 
+              <EditRecordingLabels 
+                labels={rowLabels} 
+                setLabels={setRowLabels} 
+                usePatchForm={editingMetadata}
+                patchRecordingName={props.recording.name} 
+                onPatchSubmit={handleSubmitLabelPatch}
+                onPatchCancel={handleCancelLabelPatch}
+              />
+              : rowLabels.map(l => (
+                <Text>
+                  {l.key}={l.value}
```

Comment:
This doesn't have to just be a `<Text>` element, right? If we can apply some Patternfly styling here I think it would really shine it up visually. https://www.patternfly.org/v4/components/label https://www.patternfly.org/v4/components/label-group . This would also help with the case that there are a lot of labels on a recording.

Also, in the case that the map is empty, could there be some default state here other than a simple empty table cell? Even just  a `-` character or something would be nice.